pandas/core/groupby.py

```diff
@@ -997,16 +997,16 @@ def tail(self, n=5):
         Examples
         --------
 
-        >>> df = DataFrame([[1, 2], [1, 4], [5, 6]],
+        >>> df = DataFrame([['a', 1], ['a', 2], ['b', 1], ['b', 2]],
                             columns=['A', 'B'])
         >>> df.groupby('A', as_index=False).tail(1)
            A  B
-        0  1  2
-        2  5  6
+        0  a  2
+        2  b  2
```

Comment:
I just realize in my case index will be displayed no matter as_index is False or True. In the original doc, one call to groupby doesn't have as_index=False too. Is this a bug?